iphone/Classes/TiUIClipboardProxy.m

```diff
@@ -65,18 +65,110 @@ static ClipboardType mimeTypeToDataType(NSString *mimeType)
 
 @implementation TiUIClipboardProxy
 
+-(id) init
+{
+    if (self = [super init])
+    {
+        shouldCreatePasteboard = true;
+        persistent = false;
+        isNamedPasteBoard = false;
+        isUnique = false;
```

Comment:
We should try to avoid all these helper flags. I need to go through the code in detail to check what's required. Thinkng about it, we should move all named-pasteboard-related properties to `[self pasteboard]` and check if they exist from there. If they do, we create a named one, otherwise not. Then we also don't need the manual setters and are able to configure the named pasteboard more centralized.